app/test/widgets/test_modules.dart

```diff
@@ -1,12 +1,14 @@
 import 'package:economiza_sc/stores/filters_store.dart';
```

Comment:
I made a test locally trying to use the same module as the app module and it worked. Take a look:

![image](https://github.com/jsappsbr/Economiza-SC/assets/38229718/317a8f14-4aae-4ce8-b842-6089be6b85f1)

![image](https://github.com/jsappsbr/Economiza-SC/assets/38229718/975a1b0e-bb43-4126-b51e-74ed9b1d3339)

In summary, no need to create a test module.

Reply:
Thanks! It worked, sorry man.